docs/posts/signal-number-registration-update.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-11-10
+categories:
+    - Software
+authors:
+    - matchboxbananasynergy
+tags:
+    - PSA
+    - Signal
+    - Instant Messengers
+links:
+    - Signal Configuration Guide: https://www.privacyguides.org/real-time-communication/signal-configuration-hardening/
+    - Real-Time Communication: https://www.privacyguides.org/real-time-communication/
+license: BY-SA
+---
+# Important Changes to Signal Registration and Registration Lock
+
+There has been a recent change to how Signal handles registration which primarily affects people who are using a number that they don't have exclusive access to for Signal.
```

Comment:
Signal has changed how it handles registration. This primarily affects people who are using a number that they don't have exclusive access to for Signal.